string[] computer_paths
string[] camera_paths
---
bool success
